src/material/datepicker/calendar-body.html

```diff
@@ -75,3 +76,10 @@
     </button>
   </td>
 </tr>
+
+<label [id]="_startDateLabelId" class="cdk-visually-hidden" aria-hidden="true">
```

Comment:
If this is both `cdk-visually-hidden` and `aria-hidden="true"`, we might as well just set `display: none` on it.